auth/opa/rpcauth/rpcauth.go

```diff
@@ -88,6 +94,110 @@ func NewWithPolicy(ctx context.Context, policy string, authzHooks ...RPCAuthzHoo
 	return New(p, authzHooks...), nil
 }
 
+func isMessage(descriptor protoreflect.FieldDescriptor) bool {
```

Comment:
Organizational nit: Maybe move this to a separate file? It's mostly logically separate from the rest of rpcauth.go.

Reply:
Yup makes sense. Moved it into redact.go. Can't think of better names.